R/riskdiff.R

```diff
@@ -124,6 +124,15 @@ add_risk_diff <- function(layer, ..., args=list(), distinct=TRUE) {
   assert_that(all(names(args) %in% c('p', 'alternative', 'conf.level', 'correct')),
               msg = "All arguments provided via `args` must be valid arguments of `prop.test`")
 
+
+  for (comp in comps) {
```

Comment:
If I understand correctly, the changes here are to fix a bug. In that case, is there a test to make sure the bug doesn't come back?